lib/buffered_ropes.nit

```diff
@@ -66,10 +66,9 @@ end
 #
 # Basically just returns `self` encapsulated in a `FlatString`.
 private class LeafSubstrings
-	super IndexedIterator[Text]
-
+	super IndexedIterator[FlatText]
```

Comment:
I kind of hesitate, should I let your eyes bleed ? :)

Meh, I'll change it alright !